IdentityCore/src/throttling/MSIDThrottlingService.m

```diff
@@ -112,7 +112,20 @@ + (BOOL)updateLastRefreshTimeDatasource:(id<MSIDExtendedTokenCacheDataSource>)da
                                  context:(id<MSIDRequestContext>)context
                                    error:(NSError **)error
 {
+    if([MSIDThrottlingService isThrottlingDisable])
+    {
+        return YES;
+    }
     return [MSIDThrottlingMetaDataCache updateLastRefreshTimeWithDatasource:datasource context:context error:error];
 }
 
++ (BOOL)isThrottlingDisable
+{
+#if AD_THROTTLING_DISABLED==1
```

Comment:
I think you don't technically need to do ==1. Also might be worth checking #ifdef considering this is defined in broker space.

Reply:
Addressed, thank